### PR TITLE
161226873 Authors should be able to save unpublished articles

### PR DESCRIPTION
**Pull request by emekafredy**
#### What does this PR do?
- Implement feature where authors can save unpublished articles

#### Description of Task to be completed?
- Set up reducers and actions for saving and displaying drafts in the Redux store.
- Design the viewDrafts page for all drafts by a logged in author.
- Implement feature for Authors to be able to:
	- Save article as drafts
	- Display all drafts
	- Update both published articles and drafts
	- Publish draft when ready
-N/B - The edit article button is only available if the author created the article

#### How should this be manually tested?
- clone the repo
- cd into 'thor-ah-frontend’
- checkout to the branch ```thor-ah-forntend```
- run ```npm install```
- run ```npm run dev```
- sign in with an author’s account
- click on publish to test features.
screen shot 2018-10-18 at 7 31 58 pm
screen shot 2018-10-18 at 8 05 03 pm
screen shot 2018-10-18 at 7 38 42 pm

#### What are the relevant pivotal tracker stories?
[161226873](https://www.pivotaltracker.com/story/show/161226873)

#### Questions:

### CODE REVIEW

File: src/components/ArticleContent/ArticleContent.module.scss [hunk 1]
```diff
@@ -11,3 +11,18 @@
...
+  border-radius: 4px;
+  font-size: 0.9em;
+  float: left;
+  cursor: pointer;
+}
+
+.updateLink:hover {
+  color: #fff;
+  text-decoration: none;
+}
```
Comment (emmaadesile) [inline src/components/ArticleContent/ArticleContent.module.scss]: Add new line here

File: src/actions/drafts.js [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+  payload: error,
+});
+
+// Draft action
+export const draftArticle = (articleData) => (dispatch) => {
+  dispatch(createDraft(true));
+  const { token } = localStorage;
+  return axios.post(
+    `${API}/api/articles`, articleData, {
+      headers: {
```
Comment (CEOehis) [inline src/actions/drafts.js:30]: This endpoint is it for saving draft or for creating an article?

File: src/containers/Drafts/Drafts.module.scss [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+        height: .3rem;
+        width: .3rem;
+        background-color: #000;
+        border: 50px;
+        border-radius: 50%;
+        display: inline-block;
+      }
+    }
+  }
+}
```
Comment (emmaadesile) [inline src/containers/Drafts/Drafts.module.scss]: New line required

**Review by emmaadesile — CHANGES_REQUESTED** [review]
Good job Emeka. Fix the conflicts and you are good to go

File: src/containers/Drafts/Drafts.module.scss (hunk 3 above)
Comment (emekafredy) [reply src/containers/Drafts/Drafts.module.scss]: Okay. Thanks

**Review by CEOehis — CHANGES_REQUESTED** [review]

File: src/actions/drafts.js (hunk 2 above)
Comment (emekafredy) [reply src/actions/drafts.js:30]: Yes. @CEOehis The same endpoint handles both actions

**Review by emmaadesile — APPROVED** [review]

**Review by CEOehis — APPROVED** [review]
👍 LGTM

**Review by johnkingzy — APPROVED** [review]
LGTM 👍

**Pull request merged**